.github/workflows/promote-to-stable.yml

```diff
@@ -0,0 +1,62 @@
+name: Promote
+
+on:
+  issue_comment:
+    types:
+      - created
+
+permissions:
+  issues: write
+
+jobs:
+  promote:
+    name: ⬆️ Promote to stable
+    environment: "2204 Branch"
+    runs-on: ubuntu-latest
+    if: |
+      ( !github.event.issue.pull_request )
+      && contains(github.event.comment.body, '/promote ')
+      && contains(github.event.*.labels.*.name, 'testing')
+    steps:
+      - name: ⬆️ Promote to stable
+        uses: snapcrafters/ci/promote-to-stable@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          store-token: ${{ secrets.SNAP_STORE_STABLE }}
+          snapcraft-project-root: "ffmpeg-2204-sdk"
```

Comment:
This job will get triggered on both the issue for the content snap and the SDK snap. @jnsgruk is there a way for us to figure out what issue this gets triggered on?